backend/src/main/java/de/neuefische/backend/model/SurfSpot.java

```diff
@@ -0,0 +1,18 @@
+package de.neuefische.backend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class SurfSpot {
+    private String id;
+    private String name;
+    private int longitude;
+    private int latitude;
+//    private Geography geography;
+//    private Weather weather;
+//    private Waves waves;
```

Comment:
```suggestion

```

Reply:
wanted to keep this as reminder what i might need to add in this class